cognite/client/data_classes/workflows.py

```diff
@@ -573,6 +623,49 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         }
 
 
+class SimulationTaskOutput(WorkflowTaskOutput):
+    """
+    The class represent the output of Simulation execution.
+    Args:
+        run_id (int | None): The run id of the simulation execution in the SimInt API.
+        logs (list[dict[str, Any]] | None): Logs from the simulation execution.
+        status_message (str | None): Status of the current simulation execution.
+        outputs (list[SimulatorRoutineOutput] | None): Outputs results from the simulation execution
+    """
+
+    task_type: ClassVar[str] = "simulation"
+
+    def __init__(
+        self,
+        run_id: int | None,
+        logs: list[dict[str, Any]] | None,
+        status_message: str | None,
+        outputs: list[SimulatorRoutineOutput] | None,
```

Comment:
Here as well, the type that comes out of workflow executions does not match the type that is being declared by `SimulatorRoutineOutput`:
<img width="404" alt="image" src="https://github.com/user-attachments/assets/9150f9b7-ad57-4afa-93ef-e5f6dc5844e8">

Reply:
removed the whole thing